cpp/pybind/registration/registration.cpp

```diff
@@ -79,94 +82,112 @@ class PyCorrespondenceChecker : public CorrespondenceCheckerBase {
                     "If relative change (difference) of inliner RMSE score is "
                     "lower than ``relative_rmse``, the iteration stops.")
-            .def_readwrite(
-                    "max_iteration",
-                    &registration::ICPConvergenceCriteria::max_iteration_,
-                    "Maximum iteration before iteration stops.")
-            .def("__repr__", [](const registration::ICPConvergenceCriteria &c) {
+            .def_readwrite("max_iteration",
+                           &pipelines::registration::ICPConvergenceCriteria::
+                                   max_iteration_,
+                           "Maximum iteration before iteration stops.")
+            .def("__repr__", [](const pipelines::registration::
+                                        ICPConvergenceCriteria &c) {
                 return fmt::format(
-                        "registration::ICPConvergenceCriteria class "
+                        "pipelines::registration::ICPConvergenceCriteria class "
                         "with relative_fitness={:e}, relative_rmse={:e}, "
                         "and max_iteration={:d}",
                         c.relative_fitness_, c.relative_rmse_,
                         c.max_iteration_);
             });
 
     // open3d.registration.RANSACConvergenceCriteria
-    py::class_<registration::RANSACConvergenceCriteria> ransac_criteria(
-            m, "RANSACConvergenceCriteria",
-            "Class that defines the convergence criteria of RANSAC. RANSAC "
-            "algorithm stops if the iteration number hits ``max_iteration``, "
-            "or the validation has been run for ``max_validation`` times. Note "
-            "that the validation is the most computational expensive operator "
-            "in an iteration. Most iterations do not do full validation. It is "
-            "crucial to control ``max_validation`` so that the computation "
-            "time is acceptable.");
-    py::detail::bind_copy_functions<registration::RANSACConvergenceCriteria>(
+    py::class_<pipelines::registration::RANSACConvergenceCriteria>
+            ransac_criteria(m, "RANSACConvergenceCriteria",
+                            "Class that defines the convergence criteria of "
+                            "RANSAC. RANSAC "
+                            "algorithm stops if the iteration number hits "
+                            "``max_iteration``, "
+                            "or the validation has been run for "
+                            "``max_validation`` times. Note "
+                            "that the validation is the most computational "
+                            "expensive operator "
+                            "in an iteration. Most iterations do not do full "
+                            "validation. It is "
+                            "crucial to control ``max_validation`` so that the "
+                            "computation "
+                            "time is acceptable.");
```

Comment:
clang-format splits lines, but doesn't combine lines, maybe we manually combine it into one line and let apply-style split it compactly

Reply:
fixed